mono/metadata/object.c

```diff
@@ -4326,6 +4319,13 @@ mono_unhandled_exception (MonoObject *exc)
 		if (current_appdomain_delegate)
 			call_unhandled_exception_delegate (current_domain, current_appdomain_delegate, exc);
```

Comment:
Well after reading it again, it doesn't strictly specify the order. nevermind :smile: